patches/server/0727-make-configured-features-seed-configurable.patch

```diff
@@ -0,0 +1,81 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Thonk <30448663+ExcessiveAmountsOfZombies@users.noreply.github.com>
+Date: Wed, 7 Jul 2021 18:24:24 -0500
+Subject: [PATCH] make configured features seed configurable
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+index 26e18a08a7f0bd704ff3055ce3a7814191450c85..6b0c9b11f3193fe58255af749994119f8cb50e55 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+@@ -11,6 +11,7 @@ import net.minecraft.world.entity.monster.Vindicator;
+ import net.minecraft.world.entity.monster.Zombie;
+ import com.destroystokyo.paper.antixray.ChunkPacketBlockControllerAntiXray.EngineMode;
+ import org.bukkit.Bukkit;
++import org.bukkit.configuration.ConfigurationSection;
+ import org.bukkit.configuration.file.YamlConfiguration;
+ import org.spigotmc.SpigotWorldConfig;
+ 
+@@ -176,6 +177,11 @@ public class PaperWorldConfig {
+         return config.getInt("world-settings." + worldName + "." + path, config.getInt("world-settings.default." + path));
+     }
+ 
++    private long getLong(String path, long def) {
```

Comment:
This should go into the Paper config patch

Reply:
I think I did this correctly